packages/gasket-engine/package.json

```diff
@@ -39,7 +39,6 @@
   },
   "homepage": "https://github.com/godaddy/gasket/tree/main/packages/gasket-engine",
   "dependencies": {
-    "@gasket/resolve": "^6.45.2",
     "debug": "^4.3.4"
   },
   "devDependencies": {
```

Comment:
Do we want to add typescript and the tsc typechecking scripts here yet?

Reply:
Yes, but not in this PR